document-portal/document-portal-fuse.c

```diff
@@ -260,6 +260,10 @@ typedef struct {
   char name[0];
 } XdpInvalidateData;
 
+typedef struct {
+  gboolean use_splice;
```

Comment:
Is there any need for this now? Could just replace opt->use_splice with #ifdef WITH_SPLICE ?

Reply:
This would spill `#ifdef`s across multiple places, which I find less readable.

And we might add more options in future, so I think it is better this way.